.github/workflows/main.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           - {
               name: 'Ubuntu - GCC',
               artifact: 'linux-gcc.tar.xz',
-              os: ubuntu-22.04,
+              os: ubuntu-latest-m,
```

Comment:
Hmm, is this something we really want to do? 22.04 is the current LTS, and the one we are all using at Cesium.

Reply:
`ubuntu-latest-m` is still 22.04 I believe.

I needed to switch to a large runner so that there was enough disk space for the kit debug libraries. It became too complicated to download only the release build due to a difference in how USD builds are packaged in 105.

In the near future I think we should switch back to self hosted runners with the nvidia libraries preinstalled.